### PR TITLE
docs: create overview of repos and websites for all of seedcase

### LINKED ISSUE #343: Create document listing all the different websites and products (so far)
Like some type of website directory. We can use the `gh repo` cli to create the directory using the repos and descriptions already provided on GitHub, so there is one less thing to maintain.

### PR DESCRIPTION
**Pull request by K-Beicher**
## Description

- This PR adds a PlantUML document detailing the repos and websites that we are currently working with.
- The overview is a first draft and would likely benefit from being incorporated into a text document, but we need to decide if this is the best way to display the most important information first.

## Related Issues

Closes seedcase-project/seedcase-website#343

## Reviewer Focus

This PR requires an in-depth review.

We need to look at whether this is a useful direction to go in with regards to creating an overview of our project. Or if there is a better way to keep track of our repos and websites.

Repos are denoted with 'folders' websites as documents within. Diamonds indicate that there is no website associated with the repo.

## Checklist

For general documentation:

- [x] Spell-check
    - [ ] US
    - [ ] UK
- [x] Did the page(s) preview correctly on your machine without breaking
- [x] New category words (keywords) (if any) added to the code snippet file

### CODE REVIEW

**Comment by signekb** [comment]
@K-Beicher as we talked about at the update meeting, I think the `seedcase-theme` repo is missing from the overview

**Comment by K-Beicher** [comment]
Theme has now been added, the whole thing now looks like this

Screenshot 2024-05-29 at 13 47 07

File: entries/repo-website-overview.puml
Comment (signekb) [inline entries/repo-website-overview.puml:16]: As Luke mentioned on the meeting, it might be an idea to use Mermaid as done in the SDCA research-ops repo [here](https://steno-aarhus.github.io/research-ops/) so we can add a link directly to the websites instead of writing the url?

File: entries/repo-website-overview.puml
Comment (signekb) [inline entries/repo-website-overview.puml:48]:
```suggestion
  package "seedcase-sprout" as SPROUT {
```

File: entries/repo-website-overview.puml
Comment (signekb) [inline entries/repo-website-overview.puml:11]:
```suggestion
  package seedcase-theme as THE {
```

File: entries/repo-website-overview.puml
Comment (signekb) [inline entries/repo-website-overview.puml:63]: Will/should `learn` be moved to it's own repo? It seems unintuitive that this is the only one that doesn't have its own repo?

File: entries/repo-website-overview.puml
Comment (signekb) [inline entries/repo-website-overview.puml:22]: It might be overkill to have all the sections written out here (same for the rest of the repos)? I'm thinking that it will be thing that's easy to forget if we change something in the repos? 
On the other hand, it does give an overview of the content in each of them... Not sure what my opinion is here.

File: entries/repo-website-overview.puml
Comment (signekb) [inline entries/repo-website-overview.puml:58]:
```suggestion
        History
        Team members 
```

File: entries/repo-website-overview.puml
Comment (signekb) [inline entries/repo-website-overview.puml:70]:
```suggestion
        About *-> ABOUT
        Community *--> COM
        Design *--> DES
        Decisions *--> DEC
        Learn *-> LEA
```

File: entries/repo-website-overview.puml
Comment (signekb) [inline entries/repo-website-overview.puml:62]:
```suggestion
        Learning shorts
```

File: entries/repo-website-overview.puml
Comment (signekb) [inline entries/repo-website-overview.puml:16]:
```suggestion
      object "community.seedcase-project.org" as CSP {
```

File: entries/repo-website-overview.puml
Comment (signekb) [inline entries/repo-website-overview.puml:27]:
```suggestion
      object "decisions.seedcase-project.org" as DeCP {
```

File: entries/repo-website-overview.puml
Comment (signekb) [inline entries/repo-website-overview.puml:33]:
```suggestion
      object "design.seedcase-project.org" as DeSP {
```

File: entries/repo-website-overview.puml
Comment (signekb) [inline entries/repo-website-overview.puml:34]:
```suggestion
```
Maybe not necessary to have the "Welcome" page included here? Each repo will/should have some kind of welcome/landing page and if included here, I guess we should include it for each repo in this figure?

File: entries/repo-website-overview.puml
Comment (signekb) [inline entries/repo-website-overview.puml:40]:
```suggestion
      object "team.seedcase-project.org" as TSP {
```

File: entries/repo-website-overview.puml
Comment (signekb) [inline entries/repo-website-overview.puml:43]:
```suggestion
        User tests
```

File: entries/repo-website-overview.puml
Comment (signekb) [inline entries/repo-website-overview.puml:65]:
```suggestion
      map "seedcase-project.org" as SP {
```

**Review by signekb — CHANGES_REQUESTED** [review]
Very nice! I think this is a nice overview of what we have so far. I have added a lot of casing suggestions and a comment referring to @lwjohnst86's comment during today's meeting (29/05/2024) that it might be a nice alternative to do this in Mermaid instead to include clickable links instead of writing them as text here.

File: entries/repo-website-overview.puml
Comment (K-Beicher) [reply entries/repo-website-overview.puml:22]: Same here, I went with the full option in the end, just so that we have an idea of what is inside.

File: entries/repo-website-overview.puml
Comment (K-Beicher) [reply entries/repo-website-overview.puml:63]: I'll make a discussion issue for it.

File: entries/repo-website-overview.puml
Comment (signekb) [reply entries/repo-website-overview.puml:22]: @lwjohnst86 Thoughts?

File: entries/repo-website-overview.puml
Comment (lwjohnst86) [inline entries/repo-website-overview.puml:22]: I think at this point, it is best to not include all these here, since they will quickly get outdated and this will need to updated. Referencing the website should be enough right now.

Comment (K-Beicher) [reply entries/repo-website-overview.puml:22]: Done

File: entries/repo-website-overview.puml
Comment (signekb) [reply entries/repo-website-overview.puml]: @lwjohnst86 Thoughts? Should the puml be changed to mermaid?

Comment (lwjohnst86) [reply entries/repo-website-overview.puml]: Yes please! Mermaid is easier to use to incorporate clickable links.

Comment (K-Beicher) [reply entries/repo-website-overview.puml]: The re-write in Mermaid will have to wait until after my summer holiday, do you guys want to merge this in, and then I can create an issue on converting it to Mermaid, or should I move this back into draft to make it go away from my lists?

**Comment by lwjohnst86** [comment]
I think that the roadmap has a better overview of things and acts as a document to list all our outputs and projects/products. As long as we keep that updated, that works fine for me. So I will close this PR.

**Pull request closed without merging**
